frontend/src/store/notification/reducer.ts

```diff
@@ -5,26 +5,28 @@ import { loadCurrentUserNotifications, changeStatus, getCountUncheckedUserNotifi
 
 type State = {
   dataStatus: DataStatus;
+  notificationsDataStatus: DataStatus;
   notifications: UserNotification[];
   countUncheckedNotification: number;
```

Comment:
move this `notificationCount` functionality (reducer & action) to the app reducer